assets/src/web-stories-block/block/components/storiesInspectorControls.js

```diff
@@ -170,7 +170,7 @@ const StoriesInspectorControls = (props) => {
   const previewLink = select('core/editor').getEditedPostPreviewLink();
   const carouselMessage = createInterpolateElement(
     __(
-      `<b>Note:</b> Carousel view's functionality will not work in Editor. <a>Preview</a> post to see it in action.`,
+      `<b>Note:</b> Carousel functionality will not work in Editor. <a>Preview</a> post to see it in action.`,
       'web-stories'
     ),
```

Comment:
Have we ever gotten Glider.js to work in the editor?

Reply:
I believe we had kept it on good to have priority, I was not able to work on it. But if it's something we must include then I may create a separate PR for that.

Reply:
Certainly, I'll create a new issue with the information I've.

Reply:
@swissspidy draft PR for the carousel in editor. I found some changes in my local branches/stash and was able to build on that. Check the screen grab added to see the expected diff.

https://github.com/rtCamp/web-stories-wp/pull/322